### ISSUE TITLE
address unique produces an error

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#212]
**Describe the bug**
```

(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli -c ~/configs/eos.yml address unique --columns=hostname
                                 error
0  ERROR: name 'master' is not defined
```

**Comment by jopietsch** [comment]
it's true for nxos also
```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli -c ~/configs/nxos.yml address unique --columns=hostname
                                 error
0  ERROR: name 'master' is not defined
```

**Comment by ddutt** [comment]
Fixed with commit f5b9bb75e077311e3b71671d2ccb0fe264831edd
